tavern/response.py

```diff
@@ -61,7 +64,9 @@ def __str__(self):
         else:
             return "<Not run yet>"
 
-    def _adderr(self, msg, *args, e=None):
+    def _adderr(self, msg, *args, **kwargs):
+        e = kwargs.get('e')
```

Comment:
I had to do this because the argument parsing works a little differently in 2.7. Of 4/5 ways I tried to get it going, this is the only one which works over 2&3 that I've found